docs/sidebars.js

```diff
@@ -359,13 +359,13 @@ const sidebars = {
                 "developers/contracts/writing_contracts/storage/storage_slots",
               ],
             },
-            {
-              label: "Accounts and Account Contracts",
-              type: "category",
-              items: [
-                "developers/contracts/writing_contracts/accounts/write_accounts_contract",
-              ],
-            },
+            // {
```

Comment:
check if marking the accounts page as a draft will exclude it from the sidebar as well (with this uncommented)

Reply:
Auto-exclusion from sidebar successful